lib/blob/blobstore.c

```diff
@@ -5985,6 +5985,28 @@ spdk_blob_reset_used_clusters_cache(struct spdk_blob *blob)
 		spdk_spin_unlock(&blob->bs->used_lock);
 	}
 }
+
+struct spdk_bit_array *
+spdk_blob_get_cluster_bitmap(struct spdk_blob *blob)
+{
+	struct spdk_bit_array *allocated_bitmap;
+	uint64_t i;
+
+	allocated_bitmap = spdk_bit_array_create(blob->active.num_clusters);
+	if (allocated_bitmap == NULL) {
+		SPDK_ERRLOG("Failed to allocate bit array memory for blob #%ld", blob->id);
+		return NULL;
+	}
+
+	for (i = 0; i < blob->active.num_clusters; i++) {
```

Comment:
Prefer to declare `i` in the loop as the scope is only within loop
> for (uint64_t i = 0; i < blob->active.num_clusters; i++)

Reply:
Just following the bulk of spdk code today..

Reply:
ok, they have a mix of both. You can take a call about the preference.

Reply:
Ok, I'll keep as is as the bulk of it is declared like so.